workflow-service/src/test/java/com/redhat/parodos/workflow/execution/controller/WorkFlowControllerTest.java

```diff
@@ -76,7 +80,7 @@ public void ExecuteWithValidData() throws Exception {
 	@Test
 	public void ServiceExecuteFails() throws Exception {
 		// given
-		WorkFlowRequestDTO workFlowRequestDTO = new WorkFlowRequestDTO().builder().build();
+		WorkFlowRequestDTO workFlowRequestDTO = WorkFlowRequestDTO.builder().build();
```

Comment:
Why invoke `build()` and then use setters?

Reply:
Existing code but fixed.